scripts/bluetooth/mesh/mesh_dfu_metadata.py

```diff
@@ -153,7 +153,7 @@ def from_file(cls, filename):
                 for line in config:
                     if not line.startswith("CONFIG_"):
                         continue
-                    kconfig, value = line.split("=")
+                    kconfig, value = line.split("=")[:2]
```

Comment:
Then value will contain `"AT%XMAGPIO` which is probably not what we want. 
```suggestion
                    kconfig, value = line.split("=", 1)
```